arch/sim/src/sim/up_initialstate.c

```diff
@@ -73,6 +73,10 @@ void up_initial_state(struct tcb_s *tcb)
 #else
   int ret;
 
+#if CONFIG_RR_INTERVAL > 0
+  tcb->flags |= TCB_FLAG_SCHED_RR;
```

Comment:
The change is wrong, CONFIG_RR_INTERVAL doesn't mean all thread will use the route robin policy.

Reply:
I saw that if threads don't have ```TCB_FLAG_SCHED_RR``` flag set we won't be able to execute ```sched_roundrobin_process``` in ``` nxsched_cpu_scheduler ``` file : sched/sched/sched_processtimer.c +84

Reply:
Yes, because only the thread enable roundrobin policy, then we need to execute sched_roundrobin_process. There are other policies exist(e..g FIFO or SPORADIC).
https://linux.die.net/man/2/sched_setscheduler